cli/cmd/install.go

```diff
@@ -359,6 +361,14 @@ control plane.`,
 				os.Exit(1)
 			}
 
+			shouldExist := options.identityOptions.identityExternalIssuer
+			if !options.ignoreCluster && shouldExist {
```

Comment:
Yes, we do want it to continue to work... Pff.... so now I understand what you meant when you said that there are quite few install/upgrade permutations that all need to work and lick together. So really there are only a few options to go about this. 


1. We can ditch the idea of fetching the trust anchors from the secret and always provide it via `--identity-trust-anchors-file`
2. When we have `--ignore-cluster && --identity-external-issuer` we **require** that `--identity-trust-anchors-file` is provided. 

If this is not too much cognitive luggage for the user, I would go with option 2. This will solve the problem and allow us to still benefit from fetching the trust-anchors from the secret when cluster does not need to be ignored. WDYT ?

Reply:
My thought is that the output of `--identity-external-issuer --ignore-cluster` should be the same as `--identity-external-issuer`, i.e. without the namespace and `linkerd-identity-issuer` secret, ignoring whether the namespace and secret actually exists in the cluster. We can confirm with @grampelberg tomorrow.

Reply:
Yes, I get it and agree. As I said, the two possible way I see to make this happen is to make it do we require a trustAnchor file when we have `--identity-external-issuer --ignore-cluster` because in this case we dont fail because of a "check" but because we dont have anywhere to get the trust anchors from (which lives in the secret by default).

Reply:
Per post standup discussion, we won't be supporting `linkerd install --identity-external-issuer --ignore-cluster`.